UPGRADE_NOTES.md

```diff
@@ -1,3 +1,53 @@
+## Upgrading from 0.20 to 0.21
+
+- Upgrade Lucky CLI (homebrew)
+
+```
+brew update
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#on-linux
+
+- Update versions in `shard.yml`
+  - Crystal should be `0.34.0`
+  - Lucky should be `~> 0.21.0`
+  - Authentic should be `~> 0.5.4`
+  - LuckyFlow should be `~> 0.6.3`
+
+- Run `shards update`
+
+### General updates
+
+- Rename: `config/logger.cr` to `config/log.cr`
+- Update: `config/log.cr` to use the new `Log`. [See implementation](https://github.com/luckyframework/lucky_cli/blob/master/src/web_app_skeleton/config/log.cr)
```

Comment:
There is also an update to `config/database.cr` that remove the comments and code about the query log config: 

https://github.com/luckyframework/lucky_cli/commit/1f2015eaea0883ec7b5863cc7527be344faa636a#diff-15fb0661e7cd546c3789df4bdbf83cbcL26

So they should delete the comment and if they want to log all queries they need to instead change: https://github.com/luckyframework/lucky_cli/commit/1f2015eaea0883ec7b5863cc7527be344faa636a#diff-3c1593e181f651f222dded01d5d1ab5eR35